v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-simple-component.tsx

```diff
@@ -61,17 +59,13 @@ export const UnivariateMeasureAdornmentSimpleComponent = observer(
     ) => {
       const labelSelection = select(labelRef.current)
       const labelCoords = measure.labelCoords
-      const lineHeight = 20
-      // We have to pass isGaussianFit to getLabelLinesAboveAdornment because the Gaussian fit is a special case where
-      // if there is a gaussianFit (not just a normal curve), we need to add an extra line for the adornment title.
-      const topOffset = lineHeight * (adornmentsStore?.getLabelLinesAboveAdornment(model, isGaussianFit) ?? 0)
```

Comment:
As with normal curve and standard error, getLabelLinesAboveAdornment is necessary to prevent measure label overlap